runner/ansible/roles/2.1.7/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.7'
+
+- name: "{{ test_id }}.check"
```

Comment:
I find this test quite wrong.
Here what the check should do:
1. FInd the sid
2. With the sid we can get the /usr/sap/${sid}/SYS/global/hdb/custom/config/global.ini file
3. This file is a INI file format, which has a [ha_dr_provider_SAPHanaSR] header
4. Within this header, we need to check the content of provider, path and execution order

So, could we split the check in 2 tasks, the first to get the file path and register the value.
Once we have the value, we could use the ini files task type to check the content.
What do you think?

Reply:
This should be fixed now. Was able to test with the sample file you sent me.
Still follows was was done in example code.